src/common/utils/format/format-utils.ts

```diff
@@ -44,3 +49,38 @@ export function convertBytesToKB(bytes: number | string | BigNumber, decimalPlac
 
   return bytesBN.dividedBy(BYTES_PER_KB).toFormat(decimalPlaces, BigNumber.ROUND_FLOOR);
 }
+
+export function formatBytes(bytes: number | string | BigNumber): ByteSizeResult {
+  if (bytes == null || bytes === undefined || bytes === "") {
```

Comment:
`bytes == null` already checks the undefined case. `bytes === undefined` seems redundant